src/index.html

```diff
@@ -1,12 +1,388 @@
+
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__logo"
+        >
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover"
+          class="card__buy-button"
+        >
+          Buy
+        </a>
+
+      </div>
+
+      <div class="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__logo"
+        >
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover"
+          class="card__buy-button"
+        >
+          Buy
+        </a>
+
+      </div>
+
+      <div class="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__logo"
+        >
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
```

Comment:
</div>

       <div class="card__review">Reviews: 5</div>
need space here i think.
Also you can use p here